content/posts/2018-12-05-releases.md

```diff
@@ -0,0 +1,65 @@
++++
+author = "Joseph A P Holstern"
+date = "2018-12-05T21:34:38"
+description = "2018-12-05 Weekly Release Announcements"
+draft = false
+keywords = []
+title = "2018-12-05 Weekly Release Announcements"
+topics = []
+type = "post"
+
++++
+
+Hello, this is Joseph Holsten from Sous Chefs. This is the first time I've put together a cookbook releases announcement, so please tell me what I could make better. Or stop doing terribly!
+
+## [`redisio`](https://supermarket.chef.io/cookbooks/redisio) - Major release & new maintainers!
+
+### [3.0.0](https://github.com/sous-chefs/redisio/compare/v2.7.2...v3.0.0) - 2018-11-27
+
+- This cookbook is now maintained by the Sous Chefs. If you're interested in helping with maintenance or learning more check out https://sous-chefs.org/. Thank you Brian Bianco for the work you've done over the years on this cookbook. We'll be sure to take good care of it.
+- This cookbook now requires Chef 13 or later
+- Incompatibilities with the latest selinux_policy cookbook have been resolved by using Chef's built in selinux helpers
+- All Chefstyle warnings have been resolved
+- Contributing.md and CODE_OF_CONDUCT.md files have been added
+- The build_essential resource is now directly used so that the built in resource in Chef 14+ can be used. This increases the required build-essential cookbook to 5.0+
+- Duplicate dependencies in the Berksfile have been removed
+- The chefignore file has been updated to prevent more unnecessary files from being uploaded to the chef server
+- Data bags are now loaded with the data_bag_item helper instead of Chef::EncryptedDataBagItem.load directly
+- Testing has been updated to use Delivery Local Mode which is built into ChefDK. The legacy Rakefile, Vagrantfile, Thorfile, and Cheffile have been removed
+- Platforms in Test Kitchen configurations have been updated and dokken images are now used in dokken
+
+## [`sc-chruby`](https://supermarket.chef.io/cookbooks/sc-chruby) - initial release!
+
+No really, that's all there is to be said. 
```

Comment:
What does this cookbook even do?

Reply:
fair point. also, it probably could use some real docs @damacus. What would you like me to say about it?

Reply:
at least there's the `install` resource: https://github.com/sous-chefs/sc-chruby/blob/master/resources/install.rb